File: python/easy/(62) n-mod-n.py
import sys

with open(sys.argv[1], 'r') as file:
    lines = file.readlines()
    for line in lines:
        if line:
            line = line.strip().split(',')
            n = int(line[0])
            m = int(line[1])
            print n-((n//m)*m)

File: javascript/easy/(62) n-mod-n.js
var fs  = require("fs");
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line != "") {
        var line_array = line.split(',');
        var n = parseInt(line_array[0]);
        var m = parseInt(line_array[1]);
        console.log(n-((Math.floor(n/m)*m)));
    }
});